pkg/preflight/assets/host-preflights.yaml

```diff
@@ -79,13 +79,67 @@ spec:
         backgroundWriteIOPSJobs: 6
         backgroundReadIOPS: 50
         backgroundReadIOPSJobs: 1
+    - certificate:
+        collectorName: "Kubernetes API key pair certificate"
+        exclude: '{{kurl or (not .IsPrimary) (not .IsUpgrade) }}'
```

Comment:
Could you add a more readable description of this exclusion criteria as a comment?

I think it's "when upgrading a primary node"?

Reply:
would you like to have a comment explaining all exclude rules? 
Because we use that already in other places. 

Is that a blocker to get this one merged?